elks/arch/i86/drivers/char/console-direct.c

```diff
@@ -41,34 +43,45 @@
 
 #define MAXPARMS        28
 
+#ifdef CONFIG_CONSOLE_DUAL
+#define MAX_OUTPUTS     2
+#else
+#define MAX_OUTPUTS     1
+#endif
```

Comment:
I'm still having trouble quickly understanding what an "output" is. Can you please rename MAX_OUTPUTS to MAX_DISPLAYS throughout, so that it's obvious?

We're almost there, there are still a couple unresolved conversations outstanding. Can you take a quick look at those?

Thank you!

Reply:
I applied a fixup to the initial commit, so we now have `MAX_OUTPUTS` -> `MAX_DISPLAYS` and `C->op` -> `C->display`

Reply:
Thanks, the C->op rename to C->display was a good idea too. Makes much more immediate sense to me now.